src/web/components/SubNav/SubNav.tsx

```diff
@@ -168,13 +168,15 @@ export const SubNav = ({ subNavSections, pillar, currentNavLink }: Props) => {
 
     return (
         <div
+            data-print-layout='hide'
             className={cx(
                 { [wrapperCollapsed]: collapseWrapper },
                 spaceBetween,
             )}
             data-cy="sub-nav"
         >
             <ul
+                data-print-layout='hide'
```

Comment:
if the parent is hidden, wouldnt you not need to have data-print-layout='hide' in this child?